### PR TITLE
Remove WP from human readable text in plugin

### PR DESCRIPTION
**Pull request by josephfusco**
## Description

Changes include:

- Removal of "WP" from "FaustWP" and all other references within the human readable text of the plugin
- Adds new contributors 🎉 
- Update settings menu from "Headless" to "Faust"
- Small documentation typo in `/internal/legacy.faust.org`

## Screenshots

Screen Shot 2022-10-10 at 11 11 54 AM
Screen Shot 2022-10-10 at 11 12 22 AM
Screen Shot 2022-10-10 at 12 38 11 PM

## Documentation Changes

References to "Settings -> Headless" have been updated within `/internal/faust.org`

### CODE REVIEW

File: plugins/faustwp/faustwp.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Plugin Name: FaustWP
+ * Plugin Name: Faust
```
Comment (blakewilson) [inline plugins/faustwp/faustwp.php:3]: Just to be sure, modifying the plugin name should do no harm to the WP repo right?

File: plugins/faustwp/readme.txt [hunk 2]
```diff
@@ -8,49 +8,48 @@ Requires PHP: 7.2
 License: GPLv2 or later
 License URI: https://www.gnu.org/licenses/gpl-2.0.html
 
-FaustWP transforms your traditional WordPress installation into a flexible headless CMS.
+Faust transforms your traditional WordPress installation into a flexible headless CMS.
 
 == Description ==
 
-In conjunction with Faust.js, FaustWP enables a decoupled front-end to authenticate with WordPress through GraphQL mutations and REST API endpoints. It is the connective glue between a Faust.js-powered front-end app, and a WordPress backend.
+In conjunction with the [Faust NPM packages](https://www.npmjs.com/search?q=%40faustwp), Faust enables a decoupled front-end to authenticate with WordPress through GraphQL mutations and REST API endpoints. It is the bridge between a Faust-powered front-end application, and a WordPress backend.
```
Comment (blakewilson) [inline plugins/faustwp/readme.txt:15]: 💯

**Review by blakewilson — COMMENTED** [review]
Nice work @josephfusco! Let's make sure to get a changeset in here as well.

File: plugins/faustwp/faustwp.php (hunk 1 above)
Comment (josephfusco) [reply plugins/faustwp/faustwp.php:3]: Correct. Updating "Plugin Name" for WordPress plugins will update the plugin title in WordPress,

>Screen Shot 2022-10-10 at 11 11 54 AM

as well as the plugin title on https://wordpress.org/plugins/faustwp.

>Screen Shot 2022-10-10 at 11 12 22 AM

**Review by TeresaGobble — APPROVED** [review]
Yay! Thank you so much for this hard work, Joe, LGTM. Thrilled to see my name on the list of contributors too, thank you so much for adding that for Matt and myself! 🎉

**Review by theodesp — APPROVED** [review]
LGTM!

**Pull request merged**